lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
> As for unsubscribe, my question was "does it implicitly detach too?"

No, because you're just removing a listener, and there might of course be multiple other listeners still registered. And no, we don't count the listeners because you might still want to `publish()`.